tests/test_office_version2.py

```diff
@@ -88,6 +88,35 @@ def test_post_office_name_twice(self):
         self.assertEqual(response.status_code, 409)
         self.assertEqual(response.json['error'], 'Office name already registered')
 
+    def test_get_parties(self):
```

Comment:
This methods name says `test_get_parties` yet we are testing to get offices, kindly correct that.

Reply:
> This methods name says `test_get_parties` yet we are testing to get offices, kindly correct that.



> This methods name says `test_get_parties` yet we are testing to get offices, kindly correct that.

Sorry for the typo.Let me correct that asap